tetris.py

```diff
@@ -51,8 +51,14 @@ def run_gui():
     )
     topbar_score.pack(side="left", fill="x", expand=True)
 
+    topbar_canvas_container = tkinter.Frame(topbar, bg=D_GREY, relief='ridge', height=square_size*3, width=square_size*5)
+    topbar_canvas_container.pack(side='right', expand=True)
+    topbar_canvas_container.pack_propagate(0)
+
+    #canvas_container holds topbar_canvas. This is required to prevent stretching width of the topbar_canvas as its width changes when displaying certain shapes.
```

Comment:
Split long comments to multiple lines so that they fit on the screen at once:

```suggestion
    # canvas_container holds topbar_canvas. This is required to prevent stretching
    # width of the topbar_canvas as its width changes when displaying certain shapes.
```

Also, please put this comment before `topbar_canvas_container = ...` so that I see it before I ask myself "wtf is `topbar_canvas_container`.

Reply:
Message changed and moved